src/plugins/dashboard/common/embeddable/types.ts

```diff
@@ -6,10 +6,11 @@
  * Side Public License, v 1.
  */
 
-export interface GridData {
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export type GridData = {
```

Comment:
This was done to address issues with emebedding this type inside of a `SerializableRecord` type. There are issues with index types not being present in `interface` definitions, these are not an issue with `type` definitions as they are more permissive (it seems). Otherwise we run into what I think is: https://github.com/microsoft/TypeScript/issues/1889